deploy/openshift/configmap.yaml

```diff
@@ -48,3 +49,6 @@ parameters:
   value: ""
 - name: AFFINITY
   value: "{}"
+- name: KIP_IMAGE
+  value: quay.io/eclipse/kubernetes-image-puller:next
+  
```

Comment:
@Sachin-Itagi this extra line should be removed for template sanity e.g. there should be no red marks on the PR files